api/controller.py

```diff
@@ -1157,11 +1191,23 @@ def sync(self):
 
         :return: A Response containing an OPDS feed with up-to-date information.
         """
+        patron = flask.request.patron
+
+        # Save some time if we don't believe the patron's loans or holds have
+        # changed since the last time the client requested this feed.
+        response = self.handle_conditional_request(
+            patron.last_loan_activity_sync
+        )
+        if isinstance(response, Response):
+            return response
+
+        # TODO: SimplyE used to make a HEAD request to the bookshelf feed
+        # as a quick way of checking authentication. Does this still happen?
+        # It shouldn't -- the patron profile feed should be used instead.
+        # If it's not used, we can take this out.
```

Comment:
I think it would be best to leave this in, since we may make such a request in test scripts and the like. In general, I think we should not assume that SimplyE is the only client to a CM.